react-challenge.html

```diff
@@ -41,54 +43,54 @@
         }
 
         render(){
+          let { moods, currentMood, setMood } = this.state;
           return(
             <div>
+              <Header />
               <MoodMeter
-                moodOptions={ this.state.moods[this.state.currentMood] }
+                moodOptions={ moods[currentMood] }
                 next={ this.nextMood }
                 setThisMood={ this.setThisMood }
-                set={ this.state.setMood }
+                set={ setMood }
               />
             </div>
           )
         }
       }
 
       class Header extends React.Component{
-        render(){
+        render() {
           return(
-            <div>
-              <h1>Welcome to the Mood Meter App</h1>
-            </div>
+            <h1>Welcome to the Mood Meter App</h1>
```

Comment:
only one element to render here so good thinking! consider wrapping in a `React.Fragment`, though, just in case you want to add extra elements later, and save yourself extra nodes in your HTML